go/src/koding/db/mongodb/modelhelper/deletedmember.go

```diff
@@ -0,0 +1,78 @@
+package modelhelper
+
+import (
+	"koding/db/models"
```

Comment:
i dont manage the imports, that is what comes with goimports, what are you using instead?